src/components/Dashboard/FirstSteps/index.js

```diff
@@ -14,63 +14,74 @@ import {
 } from './reducers/firstStepsReducer';
 import { UnexpectedError } from '../../shared/UnexpectedError';
 
-export const FirstSteps = InjectAppServices(({ dependencies: { systemUsageSummary } }) => {
-  const [{ firstStepsData, hasError, loading }, dispatch] = useReducer(
-    firstStepsReducer,
-    INITIAL_STATE_FIRST_STEPS,
-    initFirstStepsReducer,
-  );
+export const FirstSteps = InjectAppServices(
+  ({ dependencies: { systemUsageSummary, dopplerSystemUsageApiClient } }) => {
+    const [{ firstStepsData, hasError, loading }, dispatch] = useReducer(
+      firstStepsReducer,
+      INITIAL_STATE_FIRST_STEPS,
+      initFirstStepsReducer,
+    );
 
-  const intl = useIntl();
-  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+    const intl = useIntl();
+    const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
-  useEffect(() => {
-    const fetchData = async () => {
-      dispatch({ type: FIRST_STEPS_ACTIONS.START_FETCH });
-      const response = await systemUsageSummary.getSystemUsageSummaryData();
-      if (response.success) {
-        const { value: data } = response;
-        const dataMapped = mapSystemUsageSummary(data);
-        dispatch({
-          type: FIRST_STEPS_ACTIONS.FINISH_FETCH,
-          payload: {
-            ...dataMapped,
-            firstSteps: dataMapped.firstSteps.filter(
-              (firstStep) => firstStep.status !== UNKNOWN_STATUS,
-            ),
-          },
-        });
-      } else {
-        dispatch({ type: FIRST_STEPS_ACTIONS.FAIL_FETCH });
-      }
-    };
+    useEffect(() => {
+      // TODO: use getUserSystemUsage for the same purpose as getSystemUsageSummaryData
+      // TODO: remove systemUsageSummary service
+      const fetchData = async () => {
+        dispatch({ type: FIRST_STEPS_ACTIONS.START_FETCH });
+        const response = await Promise.all([
+          systemUsageSummary.getSystemUsageSummaryData(),
+          dopplerSystemUsageApiClient.getUserSystemUsage(),
+        ]);
+        const [systemUsageResponse, dopplerSystemUsageResponse] = response;
```

Comment:
What do you think about 

```suggestion
        const responses = await Promise.all([
          systemUsageSummary.getSystemUsageSummaryData(),
          dopplerSystemUsageApiClient.getUserSystemUsage(),
        ]);
        const [systemUsageResponse, dopplerSystemUsageResponse] = responses;
```

Or

```suggestion
        const [systemUsageResponse, dopplerSystemUsageResponse] = await Promise.all([
          systemUsageSummary.getSystemUsageSummaryData(),
          dopplerSystemUsageApiClient.getUserSystemUsage(),
        ]);
```

Reply:
It's better! 👍